<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
selection-background-color: lightblue;
background-color:#cdcdcd;
}

QWidget {
background-color: #cdcdcd;
}

QLabel {
font: 75 16pt &quot;MathJax_SansSerif&quot;;
color: fcfcfc;
}

QLabel#logoL {
background-color: #fcfcfc;
}

QPushButton {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
  border: 1px solid #ddd;
  border-radius: 3px;
  background-color: #fcfcfc;
  padding: 4px;
  color: #e5aa3d;
  line-height: 1.5em;
}
QPushButton:hover {
  color: #428bca;
  background-color: #ddd;
}

QPushButton:disabled,
QPushButton:hover:disabled,
QPushButton.disabled,
QPushButton.disabled:hover {
  background: #ededed;
  color: #888 !important;
  border-color: #ddd;
}

QLineEdit {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
color: black;
background-color: white;
selection-background-color: lightblue;
}

QFrame {
border-style: none;
}

QComboBox {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: 'white';
selection-background-color: lightblue;
}

QTableWidget {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
selection-background-color: lightblue;
}



</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>551</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>161</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Select a cut type</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>341</width>
          <height>186</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Second Point</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioButton_7">
             <property name="text">
              <string>&amp;X-Axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_8">
             <property name="text">
              <string>&amp;Y-Axis</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_8"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>First Point</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Axis of Rotation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_9"/>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_10"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Facing</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>311</width>
          <height>294</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Depth Of Cut</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Step Down</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="font">
              <font>
               <family>Cantarell</family>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>From &amp;Tool</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Overri&amp;de</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_4">
             <property name="text">
              <string>Fro&amp;m Tool</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_5">
             <property name="text">
              <string>Overr&amp;ide</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>From Too&amp;l</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_6">
             <property name="text">
              <string>Overr&amp;ide</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Spindle Speed</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Feed Rate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Offset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Pocket2D</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Volume2D</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Pocket3D</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Volume3D</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>160</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Surface3D</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_10">
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>67</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Tabs</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>521</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cut Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cutTypeCB">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>None Selected...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Registration</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Facing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Perimeter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pocket2D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Volume2D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pocket3D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Volume3D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Surface3D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tabs</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cut Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Tool</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox_4">
         <item>
          <property name="text">
           <string>None Selected</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>620</y>
        <width>166</width>
        <height>27</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>775</width>
        <height>55</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="logoL">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;Learn more at buildbotics.com!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>logo side by side.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
